### PR TITLE
 Deploy CouchDB manually

### PR DESCRIPTION
**Pull request by DanLavine**
* This gets rid of the need for the OpenWhisk configuration image 
* Update Docker images so there is no more configuration image 
* Remove specific Instructions for Minikube since everything is   deployed via yaml. The only 
  properteis that should now change is an additional environment variable in the invoker.yml 
* Update CI

### CODE REVIEW

File: kubernetes/couchdb/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+```
+kubectl apply -f couchdb.yml
+```
+
+Once the Pod is up and running, the pod then needs to be
+configured. To do this, you need to run the configuration
+pod:
+
+```
+kubectl create -f couchdb-configure.yml
```
Comment (dgrove-oss) [inline kubernetes/couchdb/README.md]: Minor nit:  The file is actually named couchdb-setup.yml

**Comment by DanLavine** [comment]
@ddragosd  I'm not sure exactly what I want to do for that. I was thinking if we want CouchDB to be HA, then you would want a different image to perform the configuration so you only need to run it once. I would think it makes it simpler to just deploy Couch and configure it once. Rather than having to build all that logic in later.

File: kubernetes/couchdb/README.md [hunk 2]
```diff
@@ -10,23 +10,33 @@ Pod. This can be done by running:
...
 ```
 
-Once the Pod is up and running, the pod then needs to be
-configured. To do this, you need to run the configuration
-pod:
+This pod goes through the process of pulling the OpenWhisk
+repo and running through some of the ansible playbooks for
+configuring CouchDB.
+
+**NOTE** the pod will say running as soon as the start command
```
Comment (ddragosd) [inline kubernetes/couchdb/README.md]: tiny typo `stay`

Comment (DanLavine) [inline kubernetes/couchdb/README.md]: oops

File: kubernetes/couchdb/docker/init.sh [hunk 3]
```diff
@@ -18,6 +24,30 @@ pushd /openwhisk
...
+
+  if [ $TIMEOUT -eq 25 ]; then
+    echo "failed to setup CouchDB"
+    exit 1
+  fi
+
+
   # setup and initialize DB
   pushd ansible
     ansible-playbook -i environments/local setup.yml \
```
Comment (ddragosd) [inline kubernetes/couchdb/docker/init.sh]: is this pulled from openwhisk master ?

File: kubernetes/couchdb/README.md (hunk 2 above)
Comment (DanLavine) [inline kubernetes/couchdb/README.md]: I'll wait for CI to make sure it passes. If it doesn't I'll update everything in 1 PR.

File: kubernetes/couchdb/docker/init.sh (hunk 3 above)
Comment (DanLavine) [inline kubernetes/couchdb/docker/init.sh]: yes. I wanted to make sure it always pulls the latest OW master code. That way I don't need to keep updating the image. Of course, this also means that if it fails to deploy then the image will need to be fixed as a whole

**Comment by DanLavine** [comment]
my updates to travis failed. Will fix those up

File: kubernetes/couchdb/docker/init.sh [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+
+  # if auth guest overwrite file
+  if [ -n "$AUTH_GUEST" ]; then
+    echo "$AUTH_GUEST" > /openwhisk/ansible/files/auth.guest
+  fi
+
+  # if auth whisk system overwrite file
+  if [ -n "$AUTH_WHISK_SYSTEM" ]; then
+    echo "$AUTH_WHISK_SYSTEM" > /openwhisk/ansible/files/auth.guest
```
Comment (dgrove-oss) [inline kubernetes/couchdb/docker/init.sh]: This should be: 

echo "$AUTH_WHISK_SYSTEM" > /openwhisk/ansible/files/auth.whisk.system

**Pull request merged**